apps/core/templates/components/header.html

```diff
@@ -10,15 +10,15 @@
           <span class="fw-bold me-2">wagtail</span>
           <span>User guide</span>
         </a>
-        <button class="bg-transparent border-0 border-start border-1 border-dark ps-4 ms-4" data-bs-toggle="modal" data-bs-target="#searchModal">
+        <button data-search-icon-button class="bg-transparent border-0 border-start border-1 border-dark ps-4 ms-4" data-bs-toggle="modal" data-bs-target="#search-modal">
           <img src="{% manifest 'images/search.svg' %}" >
         </button>
-        <div class="modal fade" id="searchModal" tabindex="-1" aria-labelledby="searchModalLabel" aria-hidden="true">
+        <div class="modal fade" id="search-modal" tabindex="-1" aria-labelledby="searchModalLabel" aria-hidden="true">
           <div class="modal-dialog modal-lg">
             <div class="modal-content rounded-3 fs-6">
               <form action="{% url 'search' %}" method="get">
                 <div class="modal-body p-4 d-flex align-items-center justify-content-center">
-                  <input type="text" name="query" placeholder="Search..." class="border border-1 border-dark bg-transparent form-control form-control-lg rounded-start w-75 ps-3">
+                  <input type="text" name="query" data-search-input placeholder="Search..." class="border border-1 border-dark bg-transparent form-control form-control-lg rounded-start w-75 ps-3">
```

Comment:
Could we add a margin between this and the search button?

Reply:
We can. I have just followed the design so I don't know if we should add a margin or not. Maybe @vossisboss and @thibaudcolas can comment on this.

Reply:
I'm not sure if Ben took the focus box into account when making the design. So I'm fine with us seeing what it looks like with some margin @Hitansh-Shah.